cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/services/AppFabricServer.java

```diff
@@ -232,4 +274,14 @@ protected void shutDown() throws Exception {
     pluginService.stopAndWait();
     privilegesFetcherProxyService.stopAndWait();
   }
+
+  private static String generateRandomPassword() {
+    /*
```

Comment:
Please use `//` per line for inline comment to be consistent with existing code base.